README.md

```diff
@@ -238,66 +295,107 @@ The following validation options would make validation pass:
 
 ```json
 {
-    "headers": 3,
-    "delimiter": ";",
-    "skip_rows": ["#"],
-    "skip_checks": ["blank-rows"]
+    "skip_errors": ["blank-row"]
+    "dialect":  {
+      "header": True,
+      "headerRows": [2],
+      "commentChar": "#",
+      "csv": {
+        "delimiter": ";"
+      }
+    }
 }
```

Comment:
Can you update the example to show some `checks` as well?

Reply:
Just did it!